src/components/SendTransaction/WarningMessages/AccountFlagged.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { InfoBlock, InfoBlockVariant } from "@stellar/design-system";
+
+export const AccountFlagged = ({ flagType = "" }) => (
```

Comment:
we could create an enum here for different flag types, but if you think it's better to allow for any type you can ignore